src/Instruments.jl

```diff
@@ -83,6 +84,6 @@ Base.show(io::IO, ::MIME"text/plain", p::Position) = print(io, amount(p), instru
 Base.zero(::Type{Position{F,A}}) where {F,A} = Position{F,A}(zero(A))
 Base.one(::Type{Position{F,A}}) where {F,A} = Position{F,A}(one(A))
 
-include(joinpath(@__DIR__,"Cash.jl"))
+include(joinpath("Cash.jl"))
```

Comment:
I'd said, include("Cash.jl"), no joinpath needed.

I think it's better just to put the code in Cash.jl back in line.
It really isn't big enough IMO to justify the bother of another file.
If it is kept, the name should be "cash.jl", as it is not a module.